wiki/sustainable_distance_education.qmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Sustainable Distance Education"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Elena Scheid"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+K (key quote):
+
+
+"An independent model may hamper students’ perception of
+current knowledge and development of collaborative competences, in contrast to the more-interactive
+collaborative and social models" (p. 10).
+
+---
+
+A (main argument):
+
+Generally, only a small influence from the particular learning model on the students’ perception of knowledge acquisition was being observed. 
```

Comment:
Please explain the main idea behind this model and how it relates to autonomy need experiences so that this argument can be understood without reading the full paper.